index.html

```diff
@@ -1,269 +1,291 @@
-                <ul class="nav-bar">
-                    <a href="">
-                        <li>Portfolio</li>
-                    </a>
-                    <a href="">
-                        <li>About</li>
-                    </a>
-                    <a href="">
-                        <li>Contact</li>
-                    </a>
-                </ul>
-            </nav>
-        </header>
-        <main>
-            <section class="heading">
-                <h1>Hey there. I’m Uzair. <br> <p>I’m a software developer</p></h1>        
-                <p class="main-text">
-                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                    don’t hesitate to contact me.
-                </p>
-            </section>
-            <aside>
-                <ul>
-                    <a href="https://github.com/muhammaduzairrazaq">
-                        <li><img src="assests/s1.png"></li>
-                    </a> 
-                    <a href="https://www.linkedin.com/in/muhammad-uzair-25479a283">
-                        <li><img src="assests/s2.png"></li>
-                    </a> 
-                    <a href="https://twitter.com/muhamma55160007?t=U6eNahc3mZKSPGDJZyqT2A&s=09">
-                        <li><img src="assests/s3.png"></li>
-                    </a>                   
-                </ul>
-            </aside>
-            <div class="icon-parent">
-                <div class="icon">
-                    <img src="assests/arrow.png" alt="down arrow">
-                </div>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/smoothscroll/1.4.10/SmoothScroll.min.js"></script>
+    <script src="https://kit.fontawesome.com/b2e81c5d42.js" crossorigin="anonymous"></script>
+    <link rel="StyleSheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <button class="menu-btn" onclick="showMenu()">
+                <i class="fa-solid fa-bars"></i>
+            </button>
+            <button class="menu-btn menu-btnn" onclick="closeMenu()">
+                <i class="fa-solid fa-xmark"></i>
+            </button>
+            <ul class="nav-bar">
+                <a href="#Portfolio" onclick="highLight('.high1')" class="margint-top">
+                    <li>Portfolio</li>
+                    <div class="high1"></div>
+                </a>
+                <a href="#About-me" onclick="highLight('.high2')">
+                    <li>About</li>
+                    <div class="high2"></div>
+                </a>
+                <a href="#Contact" onclick="highLight('.high3')">
+                    <li>Contact</li>
+                    <div class="high3"></div>
+                </a>
+            </ul>
+        </nav>
```

Comment:
[OPTIONAL] @muhammaduzairrazaq Great job adding a professional looking hamburger bar! 👍 I would suggest to manage your spacing as per Figma design and also the close button colour is a also not same as Figma 💇‍♂️ Have a Look:::


**Original Design**
![cross](https://github.com/muhammaduzairrazaq/Portfolio/assets/122353030/c6dc4138-3ab2-4675-9504-b5094fa4cdbd)

**Your design**

![yourcross](https://github.com/muhammaduzairrazaq/Portfolio/assets/122353030/e80642d0-ebd8-43cc-ba19-3a3e75d16d56)